static/src/javascripts/projects/common/modules/commercial/geo-utils.ts

```diff
@@ -0,0 +1,31 @@
+import { getCountryCode } from '../../../../lib/geolocation';
+
+// cache the users location so we only have to look it up once
+let geo: string | undefined;
```

Comment:
Just like in `geo-utils.spec.ts` we might as well use the more specific type `CountryCode`.
```suggestion
let geo: CountryCode | undefined;
```

Reply:
Updated!